glrd/manage.py

```diff
@@ -1,23 +1,30 @@
 import argparse
+import atexit
+import base64
+import fnmatch
 import json
-import pytz
-import yaml
-import boto3
+import logging
+import os
+import re
+import shutil
 import subprocess
+import sys
+import tempfile
+import time
 from datetime import datetime, timedelta
 from dateutil.relativedelta import relativedelta
-import tempfile
-import os
-import sys
-import atexit
-import shutil
-import logging
-from deepdiff import DeepDiff
+
+import boto3
+import pytz
+import yaml
 from botocore.exceptions import ClientError
+from deepdiff import DeepDiff
 from jsonschema import validate, ValidationError
-from glrd.util import *
+
 from glrd.query import load_all_releases
-import fnmatch
+from glrd.util import *
+from python_gardenlinux_lib.flavors.parse_flavors import *
+from python_gardenlinux_lib.s3.s3 import *
```

Comment:
It's always a good idea to sort by alphabet. "Always leave the code better than you found it"

Reply:
:+1: